sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java

```diff
@@ -336,28 +351,33 @@ private final synchronized void startServices(Class<? extends MobileCenterServic
             return;
         }
 
+        List<String> startedServices = new ArrayList<>();
         for (Class<? extends MobileCenterService> service : services) {
             if (service == null) {
                 MobileCenterLog.warn(LOG_TAG, "Skipping null service, please check your varargs/array does not contain any null reference.");
             } else {
                 try {
-                    startService((MobileCenterService) service.getMethod("getInstance").invoke(null));
+                    if (startService((MobileCenterService) service.getMethod("getInstance").invoke(null))) {
+                        startedServices.add(service.getSimpleName());
```

Comment:
@jaelim-ms I seen, but this function missing in `MobileCenterService` interface and cast to `AbstractMobileCenterService` seemed to me a bad idea. Should I add `getServiceName` to MobileCenterService` interface?